experimental/AlgebraicStatistics/src/DiscreteGraphicalModels.jl

```diff
@@ -0,0 +1,86 @@
+function bron_kerbosch(G, R, P, X)
+
+    cliques = []
+
+    if length(P) == 0 && length(X) == 0
+        
+        push!(cliques, R)
+    end
+
+    Q = P
+    Y = X
+
+    for v in Q
+
+        new_R = vcat(R, v);
+        new_Q = [p for p in Q if p in neighbors(G, v)]
+        new_Y = [p for p in Y if p in neighbors(G, v)]
```

Comment:
Depending on what types `Q` and `neighbors(G,v)` are, you might also be able to write
```suggestion
        nb = neighbors(G, v)
        new_Q = intersect(Q, nb)
        new_Y = intersect(Q, nb)
```